worker/peergrouper/desired.go

```diff
@@ -346,6 +280,71 @@ func (p *peerGroupChanges) possiblePeerGroupChanges(info *peerGroupInfo) {
 	logger.Debugf("assessed")
 }
 
+func isReady(status replicaset.MemberStatus) bool {
```

Comment:
if you're going to move this, I'm surprised you moved a non-member function before a member function, and then grouped the other non-member functions together. I would have expected isReady to sit next to isVotingMember.

Reply:
I moved it immediately below the only other method that uses it.
It is "Uncle Bob" Clean Code style, but I can move it back if there is precedent convention...